src/main/flight/pid.h

```diff
@@ -182,9 +182,8 @@ typedef struct pidProfile_s {
     uint8_t idle_max_increase;              // max integrated correction
     
     uint8_t ff_interpolate_sp;              // Calculate FF from interpolated setpoint
-    uint8_t ff_spread;                      // Spread ff out over at least min spread ms
```

Comment:
This needs the parameter group version to be incremented.

Reply:
done